src/ess/isissans/sans2d.py

```diff
@@ -43,31 +47,31 @@ def sample_holder_mask(
     return SampleHolderMask(holder_mask)
 
 
-def mask_detectors(
-    da: TofData[ScatteringRunType],
-    edge_mask: DetectorEdgeMask,
-    holder_mask: SampleHolderMask,
-) -> MaskedData[ScatteringRunType]:
-    """Apply pixel-specific masks to raw data.
+def to_detector_masks(
+    edge_mask: DetectorEdgeMask, holder_mask: SampleHolderMask
+) -> DetectorMasks:
+    """Gather detector masks.
+
+    Unlike :py:func:`ess.sans.masking.to_detector_mask`, this function does not rely on
+    mapping using a table of mask filenames. Instead it directly returns a dictionary
+    if multiple masks.
```

Comment:
Doesn't it always return a dict of masks, even if there is only one or no mask(s)?

Reply:
```suggestion
    of zero or more masks.
```
?

Reply:
Just "Instead it directly returns a dictionary of masks." ?